app/lib/blazer_admin_constraint.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class BlazerAdminConstraint
+  def matches?(request)
+    signin_user = UserSession.load_from_session(request.session)
+    return false if signin_user.blank?
+
+    admin_user = User.admins.kept.find_by(email: signin_user.email)
```

Comment:
[`exists?` would be cheaper here](https://api.rubyonrails.org/classes/ActiveRecord/FinderMethods.html#method-i-exists-3F)

Reply:
Thanks - have opened a new PR and updated there: https://github.com/DFE-Digital/publish-teacher-training/pull/4277/commits/7e4476256b149765ec57dfb4a058683aad519915